config/packages/security.yaml

```diff
@@ -85,6 +85,9 @@ sulu_security:
         enabled: true
     password_policy:
         enabled: true
+    two_factor:
+        force:
+            pattern: '/@sulu\.io$/'
```

Comment:
not sure if i would place this in the `force`csection. maybe it makes sens to configure it even if 2fa is not mandatory?

Reply:
@niklasnatter it allows the folowing:

```yaml
security:
    two_factor:
        force:
            pattern: '/@sulu\.io$/'

when@dev:
    security:
        two_factor:
            force:
                enabled: false
```